scripts/tasks/storybookTask.js

```diff
@@ -41,14 +42,19 @@ module.exports.buildStorybookTask = function buildStorybookTask(options) {
     quiet = options.quiet || quiet;
     ci = options.ci || ci;
 
+    const localConfigDir = path.join(process.cwd(), '.storybook');
+
     await storybook({
       mode: 'static',
       staticDir: [path.join(process.cwd(), 'static')],
-      configDir: path.join(process.cwd(), '.storybook'),
+      configDir: fs.existsSync(localConfigDir)
+        ? localConfigDir
+        : path.join(findGitRoot(), 'packages/examples/.storybook'),
       outputDir: path.join(process.cwd(), 'dist-storybook'),
       quiet,
       port: port || 3000,
       ci,
+      debugWebpack: true,
```

Comment:
Was this supposed to be checked in?

Reply:
Nope, I'm still dealing with a bunch of deploy issues and I'm using that to debug. I think it got pushed as I tried to get something working for David, but rest assured that I'll remove it by the time I merge.